tools/packaging/kernel/configs/fragments/common/base.conf

```diff
@@ -55,3 +55,7 @@ CONFIG_PERF_EVENTS=y
 # CONFIG_BLK_DEV_LOOP needs
 CONFIG_BLOCK=y
 CONFIG_BLK_DEV=y
+
+# Enable /proc/config.gz
```

Comment:
Hey the addition looks goot to me about the place,  I dont know if having it in a different fragment file `debug.conf` ? may help to semantically.  But this is not a hard requirement.

Reply:
@jcvenegas - Agreed. Branch updated.